lms/static/js/bookmarks/views/bookmarks_button.js

```diff
@@ -0,0 +1,41 @@
+;(function (define, undefined) {
+    'use strict';
+    define(['gettext', 'jquery', 'underscore', 'backbone', 'js/bookmarks/views/bookmarks_list',
+            'js/bookmarks/collections/bookmarks', 'js/views/message'],
+        function (gettext, $, _, Backbone, BookmarksListView, BookmarksCollection, MessageView) {
+
+        return Backbone.View.extend({
+
+            el: '.courseware-bookmarks-button',
+
+            loadingMessageElement: '#loading-message',
```

Comment:
Actually since these two are ids are not elements, it be nice to call them `loadingMessageElementId` and `errorMessageElementId`.